test/FluentMigrator.Tests/Integration/IntegrationTestBase.cs

```diff
@@ -178,6 +192,11 @@ private bool IsAnyServerEnabled(Predicate<Type> isMatch)
             if (!serverOptions.IsEnabled)
                 Assert.Ignore($"The configuration for {processorType.Name} is not enabled.");
 
+            var connectionString = serverOptions.ConnectionString;
+
+            if (processorType == typeof(SQLiteProcessor))
+                connectionString = $"Data Source={ _sqliteDbPath };Pooling=false;";
```

Comment:
I did try setting the connection string within app settings to `Data Source=|Data Directory|\\TestDatabase.db;Pooling=False;` and in theory SQLite should run in a mode that creates the DB if it doesn't exist but for whatever reason I just kept getting an exception that it couldn't open the database.

The only way I could get it to work was to generate a temp DB file per test, putting it in the temp directory so it gets deleted on cleanup and then overriding the default connection string.

I can see if I can move this into a custom method for IntegrationTestOptions though.

Reply:
I've tried to improve this a bit by not making it SQLite specific. The issue I had was that the `Data Directory` doesn't get updated when in a test loop so rather than just handling SQLite specific code I've implemented something that will replace the Data Directory in the connection string to point to the tests temp directory as this ensures the DB is deleted every time.

This seams to have done the trick.